annotator/static/views/rect.js

```diff
@@ -323,6 +364,10 @@ class Rect {
         return this.boundsBeforeDrag != null;
     }
 
+    isBeingDraggedViaKeydown() {
```

Comment:
Would it be possible to not have the dual functionality in the first place? Either two different flags, or rename the variable to reflect this? Or maybe a comment next to the one place this is used?